### ISSUE TITLE
Ornata v2 died

### ISSUE THREAD

**Issue by ricardoalves02** [1kc/razer-macos#427]
basically installed the razer macos software on my 2010 macbook white. it runs catalina.the lights and effect worked well except ripple. not important bcs i dont use it. i switched off the pc but instantly decided to switch it on again as i was so amazed by the lights. as i switched off i had it on wheel and the colours froze. normal i guess? stopped the wheel. but as i restast the mac the keyboard doesnt light up type and even the usb doesnt pick up on mac nor windows. so i think the software broke the keyb somehow. what can i do? im wanting to return it to the store as for a faulty unit but then i still would like to own the kwyb but dont know if the software caused the issue. i go to system management and the keyb usb doesnt shw up as i said previously.

**Comment by 1kc** [comment]
Try using razer macos to change your keyboard back to normal?

Also plugging it into a windows PC will restore everything usually.


basically installed the razer macos software on my 2010 macbook white. it runs catalina.the lights and effect worked well except ripple. not important bcs i dont use it. i switched off the pc but instantly decided to switch it on again as i was so amazed by the lights. as i switched off i had it on wheel and the colours froze. normal i guess? stopped the wheel. but as i restast the mac the keyboard doesnt light up type and even the usb doesnt pick up on mac nor windows. so i think the software broke the keyb somehow. what can i do? im wanting to return it to the store as for a faulty unit but then i still would like to own the kwyb but dont know if the software caused the issue. i go to system management and the keyb usb doesnt shw up as i said previously.

**Comment by ricardoalves02** [comment]
I can't bcs both computer systems don't recognize the keyboards USB

**Comment by 1kc** [comment]
In that case I’d probably return it. I think someone is wrong with the hardware. I doubt razer macos is capable of “bricking” it though.

On 11 Aug 2021, at 6:40 pm, ricardoalves02 ***@***.***> wrote:

﻿

I can't bcs both computer systems don't recognize the keyboards USB

**Comment by dhobi** [comment]
I agree with @1kc. Sounds like bad luck with your device. ☹️ I haven't had a single "software brick" so far and I did a lot of "wrong" stuff while developing for this project. For me it seemed like the Razer devices (I have a keyboard, mouse, mouse mat, head set, head stand) are quite forgivable in that regard.
